src/pyodide/internal/asgi.py

```diff
@@ -111,7 +115,7 @@ async def send(got):
         nonlocal headers
         if got["type"] == "http.response.start":
             status = got["status"]
-            headers = got["headers"]
+            headers = [(k.decode(), v.decode()) for k, v in got['headers']]
```

Comment:
@morgan9e I'd use double quotes here to be compliant w/ folks' formatting standards

Reply:
Was working on it!